reflex/components/graphing/recharts/charts.py

```diff
@@ -0,0 +1,191 @@
+"""A module that defines the chart components in Recharts."""
+
+from typing import Any, Dict, List, Union
+
+from reflex.vars import Var
+
+from .recharts import Recharts
+
+
+class ChartBase(Recharts):
+    """A component that wraps a victory lib."""
```

Comment:
```suggestion
    """A base class for cartesian charts in Recharts."""
```

Reply:
Yeah that's expected getting the same error as well

Reply:
I'm pretty sure I just need to disable SSR but when doing this the graphs seems to go blank, investigating

Reply:
Nikhil found this which fixed https://stackoverflow.com/questions/72311188/hydration-failed-error-using-recharts-with-nextjs